Business/UserService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Domain;
+using Domain.Interfaces;
+namespace Business
+{
+    public class UserService : IUserService
+    {
+        private readonly IUserData _userData;
+        public UserService(IUserData injectedUserData)
```

Comment:
I would like to keep the param name as just userData if you agree

Reply:
I agree